File: app/src/main/java/com/neome/api/meta/base/dto/EntVdBranchIfElse.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.EnumDefnKindBranchIfElse;
import com.neome.api.meta.base.Types.MetaIdVar;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class EntVdBranchIfElse extends EntVdAutoStep
{
  @Nullable
  public StudioMapOfCondition condition;

  @Nullable
  public MetaIdVar conditionVarId;

  @Nullable
  public EnumDefnKindBranchIfElse option;
}

File: app/src/main/java/com/neome/api/meta/base/dto/EntVdBranchIfElse.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnKindBranchIfElse
import com.neome.api.meta.base.Types.MetaIdVar

open class EntVdBranchIfElse : EntVdAutoStep() {
    var condition: StudioMapOfCondition? = null
    var conditionVarId: MetaIdVar? = null
    var option: EnumDefnKindBranchIfElse? = null
}
